pages/osx/carthage.md

```diff
@@ -0,0 +1,23 @@
+# carthage
+
+> Carthage is a dependency management tool for Swift and Objective-C.
```

Comment:
The dependency manager says that its a package manager for "your Cocoa application". I think that's a better wording that "for Swift and Objective-C". I initially thought it has something do with the programming language itself. Then realized its a package manager for projects of those languages.

It might be helpful to make that very clear to avoid any confusion.